cfg/qt.cfg

```diff
@@ -59,8 +59,8 @@
   <reflection>
     <call arg="2">invokeMethod</call>
   </reflection>
-  <define name="SIGNAL(X)" value="#X"/>
-  <define name="SLOT(X)" value="#X"/>
+  <!-- <define name="SIGNAL(X)" value="#X"/> -->
+  <!-- <define name="SLOT(X)" value="#X"/> -->
```

Comment:
Won't this cause `[checkLibraryFunction](http://cppcheck1.osuosl.org:8000/check_library_function_report.html)` warnings? Can we define `value="X"`?